### PR TITLE
Bumpmap Cleanups

### PR DESCRIPTION
**Pull request by Hoikas**
This changeset fixes the traceback introduced by PyHSPlasma changes by moving bumpmap UV insertion into the main loop. I further took the liberty to make the code more friendly to The Zen of Korman.

### CODE REVIEW

File: korman/exporter/mesh.py [hunk 1]
```diff
@@ -267,22 +288,17 @@ def _export_geometry(self, bo, mesh, materials, geospans):
                         geoVertex.normal = hsVector3(*tessface.normal)
 
                     geoVertex.color = hsColor32(*vertex_color)
-                    geoVertex.uvs = [hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws]
+                    uvs = list([hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws])
+                    if bumpmap is not None:
+                        uvs.append(dPosDu)
+                        uvs.append(dPosDv)
```
Comment (dpogue) [inline korman/exporter/mesh.py]: This is slightly different than the original code: In the original, we loop through and each vertex can have multiple UV gradients that are summed and then normalized. In this code, you're just appending already-normalized UV gradients to a list.

Comment (Hoikas) [reply korman/exporter/mesh.py]: oh! thanks, that makes sense now. I still don't like the idea of having to convert the UVs from a tuple to a list over and over and over again, but so be it, I guess...

**Comment by Hoikas** [comment]
Updated to fix issues from review

**Review by dpogue — APPROVED** [review]
:+1:

**Pull request merged**